docs/_sidebar.md

```diff
@@ -2,6 +2,7 @@
 
   - [Installing Eva](installing_eva.md)
   - [Using Eva](using_eva.md)
+  - [using Eva With DA Monitor] (using_eva_DA_monitor.md)
```

Comment:
Let's call it EMC DA Monitor

Reply:
Just for the sidebar, for clarity

Reply:
So "using Eva with EMC DA Monitor" in the sidebar, right?

Reply:
Yeah

Reply:
Done.